### PR TITLE
Fix EDGE-T mapping issues

### PR DESCRIPTION
**Pull request by johannah-pik**
## Purpose of this PR
Fix mapping issues reported [here](https://github.com/pik-piam/edgeTransport/issues/244):
- ES piam_unit was changed from bn to billion as it is reported in reporttransport and adjusted in remind2 accordingly. Es we report more variables with spelling out "billion" (e.g. Capital) I changed it in that direction
- The name of FE|Transport|Pass|Road|LDV|Two-Wheelers was corrected to FE|Transport|Pass|Road|LDV|Two Wheelers
- In the AR6 mapping ES|Transport|VKM|Road and FE|Transport|Freight|Liquids|Oil were removed as they are not reported by EDGE-T
- Piam_units for the stock variables were adjusted to million veh
- In the NAV mapping FE|Transport|LDV|Electricity was changed to FE|Transport|Pass|Road|LDV|Electricity (similar for Gases and Hydrogen)

## Checklist:
- [x ] I did not use Excel to open csv files or checked that no side-effects occur (changed values, many new quotation marks, …)
- [ ] I added any renamed piam_variable to [`renamed_piam_variables.csv`](https://github.com/pik-piam/piamInterfaces/blob/master/inst/renamed_piam_variables.csv) to guarantee backwards compatibility.

It is recommended to have a look at the [tutorial](https://github.com/pik-piam/piamInterfaces/blob/master/tutorial.md) before submission.

### CODE REVIEW

**Comment by johannah-pik** [comment]
I dont really know what [renamed_piam_variables.csv](https://github.com/pik-piam/piamInterfaces/blob/master/inst/renamed_piam_variables.csv) does.. Am I supposed to add anything here as these were only bug fixes?

File: inst/mappings/mapping_NAVIGATE.csv [hunk 1]
```diff
@@ -169,25 +169,25 @@ idx;Tier;Category;variable;unit;piam_variable;piam_unit;piam_factor;internal_com
...
 140;1;economy;Value Added|Industry;billion US$2010/yr;;;;;;value added of the industry sector;
-141;1;economy;Capital Formation;billion US$2010/yr;Investments;billion US$2005/yr;1.12;;;net additions to the physical capital stock;R
+141;1;economy;Capital Formation;billion US$2010/yr;Investments;billion US$2005/yr;1,12;;;net additions to the physical capital stock;R
 142;1;economy;Expenditure|Government;billion US$2010/yr;;;;;;total government expenditure;
 143;1;economy;Expenditure|Household;billion US$2010/yr;;;;;;Total household expenditure;
 144;1;economy;Export;billion US$2010/yr;;;;;;total exports measured in monetary quantities.;
 145;1;economy;Import;billion US$2010/yr;;;;;;total imports measured in monetary quantities.;
 146;2;economy;Capital Stock;billion US$2010/yr;;;;;;Macroeconomic capital stock;
 147;2;economy;Interest Rate|Real;%;Interest Rate t/(t-1)|Real;unitless;100;;;Real interest rate or return on capital that is relevant for energy system investments;R
-148;2;economy;Value Added|Industry|Chemicals;billion US$2010/yr;Value Added|Industry|Chemicals;billion US$2005/yr;1.12;;;value added of the chemical industries.;R
```
Comment (orichters) [inline inst/mappings/mapping_NAVIGATE.csv:185]: Can you please undo all the adjustments of `1.12` to `1,12` with a comma? This small script should do it:
```
devtools::load_all(); library(tidyverse)
write.csv2(getMapping("NAVIGATE") %>% mutate(piam_factor = gsub(",", ".", .data$piam_factor)), mappingNames("NAVIGATE"), na = "", row.names = FALSE, quote = FALSE)
```

**Comment by orichters** [comment]
> I dont really know what [renamed_piam_variables.csv](https://github.com/pik-piam/piamInterfaces/blob/master/inst/renamed_piam_variables.csv) does.. Am I supposed to add anything here as these were only bug fixes?

You should add cases where a variable was renamed, but it might still be that people come along with data containing the old variable. I just scrolled through your files and I think adding those might be good:
```
FE|Transport|Pass|Road|LDV|Two-Wheelers;FE|Transport|Pass|Road|LDV|Two Wheelers
FE|Transport|Pass|Road|LDV|Electricity;FE|Transport|LDV|Electricity
FE|Transport|Pass|Road|LDV|Gases;FE|Transport|LDV|Gases
FE|Transport|Pass|Road|LDV|Hydrogen;FE|Transport|LDV|Hydrogen
```

**Comment by orichters** [comment]
Can you please have a look at https://github.com/pik-piam/piamInterfaces/pull/361/files#diff-831dafe67a14f466370eae61279d3b9a31a5283b5bce57b24551862bac837226L880-L881, that seems wrong to me.

```
Final Energy|Transportation (w/ bunkers);EJ/yr ... FE|Transport with bunkers;EJ/yr
Final Energy|Transportation (w/ bunkers);EJ/yr ... FE|Transport;EJ/yr
```
I think you don't need both, right?

**Comment by johannah-pik** [comment]
Thanks Oli! And sorry for surrendering on this end after the bunkers discussion.. 
I removed the changes in [renamed_piam_variables.csv](https://github.com/pik-piam/piamInterfaces/blob/master/inst/renamed_piam_variables.csv) that are only valid for edgeT and not for the remind2 variables that should be used in the project reportings. 
The adjustments in this PR are bugfixes and not traceable name changes..

**Comment by johannah-pik** [comment]
Do you know what is meant by the abbreviations in the source column?

**Comment by orichters** [comment]
-> https://github.com/pik-piam/piamInterfaces/blob/master/tutorial.md#mappings

Johanna, I am a bit sorry to say that, but @laurinks [just added the ScenarioMIP template](https://github.com/pik-piam/piamInterfaces/pull/370/files), and copied several aspects from NAVIGATE, so it might be necessary to copy changes to this new mapping as well. I'm not saying you have to do that, but maybe you can quickly coordinate with each other?

**Comment by laurinks** [comment]
Thanks, @orichters, to point this out! @johannah-pik: I am PIK tomorrow if you briefly want to coordinate to see what the most efficient way to copying the changes into the mapping_ScenarioMIP would be (?).

**Comment by johannah-pik** [comment]
I have a script to apply the changes that should be relatively easy to adjust for the ScenarioMIP mapping. I will forward it to you on mattermost @laurinks. And I can offer to review the transport variable mapping in the end. 
If it doesn't work as expected, I am at PIK and we could also quickly meet and discuss :)

**Review by orichters — APPROVED** [review]
Fine from my side. I suggest you merge soon, and one can still adjust things if needed, but we are at least getting rid of some major issues and can use next week-ends automated model tests to investigate whether we fixed everything.

**Pull request merged**